### DISCUSSION TITLE
你好，有四个接口无法使用

### DISCUSSION THREAD

**Discussion by laoshiren0822** [CefView/QCefView#60] [General]
CefViewBrowserClientDelegateInterface抽象类中定义的是 const std::string &，但是在CCefClientDelegate接口实现用的是const CefString&

CefViewBrowserClientDelegateInterface声明：
   virtual void addressChanged(CefRefPtr<CefBrowser>& browser, int64_t frameId, const std::string& url) = 0;

   virtual void titleChanged(CefRefPtr<CefBrowser>& browser, const std::string& title) = 0;

   virtual bool tooltipMessage(CefRefPtr<CefBrowser>& browser, const std::string& text) = 0;

   virtual void statusMessage(CefRefPtr<CefBrowser>& browser, const std::string& value) = 0;

CCefClientDelegate继承CefViewBrowserClientDelegateInterface
   virtual void addressChanged(CefRefPtr<CefBrowser>& browser, int frameId, const CefString& url);

  virtual void titleChanged(CefRefPtr<CefBrowser>& browser, const CefString& title);

  virtual bool tooltipMessage(CefRefPtr<CefBrowser>& browser, const CefString& text);

  virtual void statusMessage(CefRefPtr<CefBrowser>& browser, const CefString& value);

**Comment by tishion** [comment]
你是不是用了比较老的代码？这里已经统一了。
